src/components/Main.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from "react";
+import MoviesErrorBoundary from "./MoviesErrorBoundary";
+import MoviesList from './MoviesList'
+
+const category = [
+    'All', 'Documentary', 'Comedy', 'Horror', 'Crime'
+]
+
+const filters = [
+    'release date', 'best rating'
+]
+
+const Main: React.FC = () => {
+    const [select, setSelect] = useState('release date')
+    
+    const listCategory = category.map((cat: string) => <li key={cat}>{cat}</li>)
+
+    const listFilters = filters.map((filter: string, i) => {
+        return <option key={i} value={filter}>{filter.toUpperCase()}</option>
```

Comment:
Maybe it's better to use `filter` for `key`?

Reply:
fixed it